cypress/support/fragments/data_import/logs/logs.js

```diff
@@ -50,4 +51,24 @@ export default {
   },
 
   quantityRecordsInInvoice,
+
+  goToTitleLink: (title) => {
+    // When you click on a link, it opens in a new tab. Because of this, a direct transition to the link is carried out.
+    cy.do(
+      MultiColumnList({ id: 'search-results-list' })
+        .find(Link(title)).perform(element => {
+          cy.visit(element.href);
+        })
+    );
+  },
+
+  checkAuthorityLogJSON: () => {
+    cy.do(Button('Authority').click());
+    cy.expect([
+      HTML('"sourceFileId":').exists(),
```

Comment:
Don't use naked HTML pls
It will make a lot of difficulties in debugging
Try to use HTML after another any selector like this

TestField().find(HTML())

Reply:
OK, I understand. But in this case, I had to use only HTML, since the page is a textual representation of JSON.
```
<code class="info---bee3O">{
  <span class="key---wzTh6">"id":</span> <span class="string---LFT5x">"d47173e9-d5e8-40bd-b785-e89d34b5342c"</span>,
  <span class="key---wzTh6">"snapshotId":</span> <span class="string---LFT5x">"1cbfbaaf-0980-4ed3-9eaa-1cb6492d2cf8"</span>,
```